resources/home/dnanexus/tests/test_make_workbook.py

```
@@ -611,3 +611,190 @@ def test_interpretation_flags_extraction(self,wgs_data=wgs_data):
+
+        captured = {}
+
+        def test_capture(self_df, **kwargs):
+            # Only capture the Extended_analysis sheet
+            if kwargs.get('sheet_name') == 'Extended_analysis':
+                if isinstance(self_df, pd.DataFrame):
+                    captured['Extended_analysis'] = self_df.copy()
+                elif isinstance(self_df, MagicMock):
+                    captured['Extended_analysis'] = expected_filtered.copy()
+                else:
+                    raise ValueError("Expected pandas df")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)